containerless/rust/controller-agent/src/controller/compiler.rs

```diff
@@ -131,6 +131,15 @@ pub fn dispatcher_deployment_spec(version: usize) -> k8s::Deployment {
                         )
                         .build(),
                 )
+                .strategy(DeploymentStrategyBuilder::new()
+                    .type_(DeploymentStrategyType::RollingUpdate)
+                    .rolling_update(RollingUpdateDeploymentBuilder::new()
```

Comment:
I was under the impression that the rolling update is the default. Is that not the case?